frontend/test/metabase/scenarios/visualizations/reproductions/16170-line-mongo-replace-missing-values.cy.spec.js

```diff
@@ -34,7 +34,7 @@ describe("issue 16170", () => {
       cy.get(".dot").eq(-2).trigger("mousemove", { force: true });
 
       popover().within(() => {
-        testPairedTooltipValues("Created At", "2018");
+        testPairedTooltipValues("Created At", "2019");
```

Comment:
I'm not quite sure how this test works, but the value `"6.578"` belongs to year `2019`.

Reply:
@losrebellos Could you please have a look at this change?

Reply:
It seems that this specific test is checking what appears in the Tooltip.
Does it look correct @nemanjaglumac?

Reply:
I'm testing this locally and the test fails for me. The previous values seem to be correct.
![image](https://user-images.githubusercontent.com/31325167/182927051-cd5ecb49-2bc0-471c-b9c5-df305c145d83.png)

Now it gets even more confusing because @metamben says that 6578 belongs to 2019. In my case, it belongs to 2018 and 2019 is the last year - not 2020.

I'm really confused but the test seems to be passing in CI...

Reply:
I can definitely see orders with `created_at` in the sample database.